daemon/src/payout_curve.rs

```diff
@@ -109,9 +108,9 @@ fn calculate_payout_parameters(
 
             let long_amount = to_sats(long_amount_btc)?;
```

Comment:
If you move this line further down, you can avoid converting between `Amount` and `u64`.

Reply:
Ah fair point. Can we change `to_sats` to return `Amount` instead?